test/kafka-resumption/setup.td

```diff
@@ -14,19 +14,10 @@ $ http-request method=POST url=http://toxiproxy:8474/proxies content-type=applic
   "upstream": "kafka:9091"
 }
 
-$ file-append path=dynamic.csv
-city,state,zip
-Rochester,NY,14618
```

Comment:
hmmmmmm i wonder who wrote this test....

Reply:
Funnily enough, not me! https://github.com/MaterializeInc/materialize/blame/main/test/kafka-resumption/setup.td